src/components/molecules/DetailTitle/index.tsx

```diff
@@ -3,13 +3,13 @@ import React from 'react';
 import Styled from 'styled-components';
 
 interface IProps {
-  speaker?: string;
+  authorNickname?: string;
   interest?: string[];
   createdAt?: string;
   title?: string;
 }
 
-function DetailTitle({ title, speaker, createdAt, interest }: IProps): React.ReactElement {
+function DetailTitle({ title, authorNickname, createdAt, interest }: IProps): React.ReactElement {
```

Comment:
```suggestion
function DetailTitle({ title, authorNickname, createdAt, interestList }: IProps): React.ReactElement {
```
복수형이니까 이렇게 하는게 맞을 듯 합니다!